### PR TITLE
Add bit-shift operators for supporting hashing

### PR DESCRIPTION
**Pull request by wschin**
There are two essential operators used in many hashing algorithms. This PR proposes to add [bit-shift](https://en.wikipedia.org/wiki/Bitwise_operations_in_C#Shift_operators) to cover them. Here is reference to a popular [hashing algorithm](https://en.wikipedia.org/wiki/MurmurHash#Implementations).

### CODE REVIEW

File: onnx/defs/logical/defs.cc [hunk 1]
```diff
@@ -138,4 +138,68 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrains input/output to boolean tensors.")
         .TypeAndShapeInferenceFunction(unaryLogicalOpInference));
 
+static const char* LeftShift_ver10_doc = R"DOC(
```
Comment (linkerzhang) [inline onnx/defs/logical/defs.cc]: I think it's better to have on op to current both case (right and left).

File: onnx/defs/logical/defs.cc [hunk 2]
```diff
@@ -138,4 +138,68 @@ ONNX_OPERATOR_SET_SCHEMA(
...
+ binary representation toward the right side so that the input value 
+ is decreased. The input X is the tensor to be shifted while another
+ input Y specifies the amounts of shift. For example, if X and S are
+ [1, 4] and [1, 1], respectively, then the corresponding output Z
+ would be [0, 2]. Because this operator supports Numpy-style broadcasting,
+ X's and Y's shapes are not necessarily identical.
+
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
```
Comment (linkerzhang) [inline onnx/defs/logical/defs.cc]: please also add test case for the op and also shape inference test.

**Comment by linkerzhang** [comment]
run https://github.com/onnx/onnx/blob/master/onnx/backend/test/stat_coverage.py to update the test coverage doc (that's why Linux ci failed). meanwhile, a trick when running this script, you'd have to explicitly set ML=false to update the right doc. https://github.com/onnx/onnx/blob/master/onnx/backend/test/stat_coverage.py#L232

I know this kind of doc update experience is bad in windows, it should be improved (some unified script should be created for it).

File: onnx/defs/logical/defs.cc (hunk 2 above)
Comment (wschin) [reply onnx/defs/logical/defs.cc]: Sure.

File: onnx/defs/logical/defs.cc (hunk 1 above)
Comment (wschin) [reply onnx/defs/logical/defs.cc]: No problem.

**Comment by wschin** [comment]
> run https://github.com/onnx/onnx/blob/master/onnx/backend/test/stat_coverage.py to update the test coverage doc (that's why Linux ci failed). meanwhile, a trick when running this script, you'd have to explicitly set ML=false to update the right doc. https://github.com/onnx/onnx/blob/master/onnx/backend/test/stat_coverage.py#L232
> I know this kind of doc update experience is bad in windows, it should be improved (some unified script should be created for it).

Just tried some combinations of reinstall/ONNX_ML=0 and none of them worked..

**Pull request merged**